rpm/kubelet.spec

```diff
@@ -54,7 +54,7 @@ Source7: https://github.com/kubernetes-incubator/cri-tools/releases/download/v%{
 BuildRequires: systemd
 BuildRequires: curl
 Requires: iptables >= 1.4.21
-Requires: kubernetes-cni = %{CNI_VERSION}
+Requires: kubernetes-cni >= %{CNI_VERSION}
```

Comment:
because this spec is used across releases this needs to float to the lowest common denominator and explicitly be >= 0.6.0